docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
+| sideEffects | Provides access to the saga's side effect functions.                              |
+
+In addition to user-defined side effect functions, the `SideEffects` object contains the following default side effects:
+
+| Function Name                       | Description                                                                                 |
+| ----------------------------------- | ------------------------------------------------------------------------------------------- |
+| [executeCommand](#executecommand)   | Sends a command with the specified payload to an aggregate.                                 |
+| [scheduleCommand](#schedulecommand) | Similar to `executeCommand`, but delays command execution until a specified moment in time. |
+
+The `sideEffects` object's `isEnabled` field indicates whether or not side effects are enabled for the saga.
+
+### executeCommand
+
+Sends a command with the specified payload to an aggregate.
+
+#### Arguments
+
+| Argument Name | Description                                                                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| command       | Specifies a command object. Refer to the [Write Side](write-side#sending-a-command) document for more information. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/saga-sample/saga.js#execute)
+```js
+await sideEffects.executeCommand({
+  aggregateName: 'User',
+  aggregateId: event.aggregateId,
+  type: 'requestConfirmUser',
+  payload: event.payload
+})
+```
+
+<!-- prettier-ignore-end -->
+
+### scheduleCommand
+
+Similar to `executeCommand`, but delays command execution until a specified moment in time.
+
+#### Arguments
+
+| Argument Name | Description                                                                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------ |
+| command       | Specifies a command object. Refer to the [Write Side](write-side#sending-a-command) document for more information. |
+
+#### Example
+
+<!-- prettier-ignore-start -->
+
+[mdis]:# (../tests/saga-sample/saga.js#schedule)
+```js
+await sideEffects.scheduleCommand(
+  event.timestamp + 1000 * 60 * 60 * 24 * 7,
+  {
+    aggregateName: 'User',
+    aggregateId: event.aggregateId,
+    type: 'forgetUser',
+    payload: {}
+  }
+)
+```
+
+<!-- prettier-ignore-end -->
+
+## Client-Side API
+
+### HTTP API
+
+Resolve provides a standard HTTP API that allows you to send aggregate commands and query Read Models and View Models. Refer to the [Standard HTTP API](curl.md) document for more information.
+
+#### Read Model API
+
+You can query a Read Model from the client side by sending a POST request to the following URL:
```

Comment:
and here

Reply:
'To query a Read Model from the client side, send a POST request to the following URL:'

Reply:
Ok